source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -275,8 +275,8 @@ void RedisCluster::RedisDiscoverySession::startResolveRedis() {
   if (parent_.hosts_.empty()) {
     const int rand_idx = parent_.random_.random() % discovery_address_list_.size();
     auto it = discovery_address_list_.begin();
-    std::next(it, rand_idx);
-    host = Upstream::HostSharedPtr{new RedisHost(parent_.info(), "", *it, parent_, true)};
+    host = Upstream::HostSharedPtr{
+        new RedisHost(parent_.info(), "", *std::next(it, rand_idx), parent_, true)};
```

Comment:
@htuch - this is the only non-build, non-test change in this entire patchset. Would you like me to break it out for you as an independently reviewed PR, or are we ok to merge and iterate at this point? It's the remaining example of a discarded no-op result in the //source/... since I re-merged.

Reply:
No, I was not asking for this. I was pointing out that the test changes alone could have been upstreamed independently. Anyway, NBD.